packages/block-editor/src/components/global-styles/typography-panel.js

```diff
@@ -260,15 +270,19 @@ export default function TypographyPanel( {
 	const hasTextDecoration = () => !! value?.typography?.textDecoration;
 	const resetTextDecoration = () => setTextDecoration( undefined );
 
-	const resetAll = () => {
-		onChange( {
-			...value,
+	const resetAllFilter = useCallback( ( previousValue ) => {
+		return {
+			...previousValue,
 			typography: {},
-		} );
-	};
+		};
+	}, [] );
 
 	return (
-		<Wrapper resetAll={ resetAll }>
+		<Wrapper
```

Comment:
Just noticed, and this is possibly related to the other comments from @ciampo and @aaronrobertshaw, but this `Wrapper` appears to be missing passing in a `panelId`.

Reply:
Actually, this is not really needed in all of our usage of the as prop.